### DISCUSSION TITLE
\Possible impact of changing Cassandra table compaction strategy

### DISCUSSION THREAD

**Discussion by xhyzzZ** [jaegertracing/jaeger#5268] [Q&A]
### Requirement

From the Cassandra creation code:
dependencies_v2: https://github.com/jaegertracing/jaeger/blob/main/plugin/storage/cassandra/schema/v003.cql.tmpl#L202
operation_names_v2: https://github.com/jaegertracing/jaeger/blob/main/plugin/storage/cassandra/schema/v003.cql.tmpl#L102
service_names: https://github.com/jaegertracing/jaeger/blob/main/plugin/storage/cassandra/schema/v003.cql.tmpl#L86

I can see these tables are using `org.apache.cassandra.db.compaction.SizeTieredCompactionStrategy`. Is there any reason why we are not using `LCS (Leveled Compaction Strategy)`? What will be the impact of changing this compaction strategy by ourselves? 

### Problem

Would like to change the compaction strategy as suggested by our DevOps team.

**Comment by yurishkuro** [comment]
If you have DevOps team who maintains Cassandra, they will have much deeper knowledge and understanding of the impact than trying to get this answered here.
